### PR TITLE
Fix for showing correct name for operators in operator hub

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
fixes https://jira.coreos.com/browse/CONSOLE-1217

### CODE REVIEW

File: frontend/public/components/operator-hub/operator-hub-page.tsx [hunk 1]
```diff
@@ -18,59 +18,36 @@ const normalizePackageManifests = (packageManifests: PackageManifestKind[] = [],
...
-    const catalogSource = _.get(packageManifest, 'status.catalogSource');
-    const catalogSourceNamespace = _.get(packageManifest, 'status.catalogSourceNamespace');
-    const providerType = getOperatorProviderType(packageManifest);
     return {
       obj: packageManifest,
       kind: PackageManifestModel.kind,
-      name,
-      uid,
+      name: _.get(currentCSVDesc, 'displayName', packageManifest.metadata.name),
+      uid: `${packageManifest.metadata.name}/${packageManifest.status.catalogSourceNamespace}`,
```
Comment (spadgett) [inline frontend/public/components/operator-hub/operator-hub-page.tsx]: I can't remember if this came up before, but is it possible to use `packageManifest.metadata.uid`?

File: frontend/public/components/operator-hub/operator-hub-page.tsx [hunk 2]
```diff
@@ -18,59 +18,36 @@ const normalizePackageManifests = (packageManifests: PackageManifestKind[] = [],
...
-      containerImage,
-      createdAt,
-      support,
-      longDescription,
-      categories: categoryArray,
-      catalogSource,
-      catalogSourceNamespace,
+      imgUrl: iconObj ? `data:${iconObj.mediatype};base64,${iconObj.base64data}` : operatorImg,
+      description: currentCSVAnnotations.description || currentCSVDesc.description,
+      longDescription: currentCSVDesc.description || currentCSVAnnotations.description,
```
Comment (spadgett) [inline frontend/public/components/operator-hub/operator-hub-page.tsx]: It's unclear to my why in one place we prefer `currentCSVAnnotations` and in other we prefer `currentCSVDesc`

File: frontend/public/components/operator-hub/operator-hub-page.tsx (hunk 1 above)
Comment (jeff-phillips-18) [reply frontend/public/components/operator-hub/operator-hub-page.tsx]: There is no uid in the package manifest's metadata.

File: frontend/public/components/operator-hub/operator-hub-page.tsx (hunk 2 above)
Comment (jeff-phillips-18) [reply frontend/public/components/operator-hub/operator-hub-page.tsx]: @galletti94, I'm pulling data from where I can get it. Is there an explanation of these fields and why the short description is in the annotations and the long is in the CSV desc?

Comment (galletti94) [reply frontend/public/components/operator-hub/operator-hub-page.tsx]: The way I see it is anything in the CSV desc are fields that are well defined in the CSV crd and are expected to be there when someone writes a CSV. Annotations is an additional field for anything extra that we need for pretty rendering or metadata. The hope is to one day have everything part of the former

**Comment by jeff-phillips-18** [comment]
@spadgett Do you want me to add comments to the code explaining the above?

**Pull request merged**